pkg/operator/certrotation/signer.go

```diff
@@ -67,6 +67,7 @@ func (c RotatedSigningCASecret) EnsureSigningCertKeyPair(ctx context.Context) (*
 	if err != nil && !apierrors.IsNotFound(err) {
 		return nil, false, err
 	}
+	var signerExists = true
```

Comment:
This is required to pass unit tests, which do delete+create (and don't set ResourceVersion afterwards). It would be reworked after `ApplySecretDoNotUse` is removed